roles/openshift_facts/library/openshift_facts.py

```diff
@@ -623,7 +623,7 @@ def set_deployment_facts_if_unset(facts):
             service_type = 'atomic-openshift'
             if deployment_type == 'origin':
                 service_type = 'origin'
-            elif deployment_type in ['enterprise', 'online']:
+            elif deployment_type in ['enterprise']:
```

Comment:
Would it be sufficient to wait until all our 3.0 clusters are upgraded before merging this? As of this week, we won't be managing any 3.0 online clusters, so I don't think we'll need the old service_type again. All the clusters will be using the atomic-openshift* service names. I checked with Wesley and he confirmed we'll be 100% on version 3.1 after the upgrades.